package.json

```diff
@@ -39,7 +40,7 @@
     "npm": "=5.0.3",
     "pg-monitor": "=0.8.2",
     "pg-native": "=1.10.1",
-    "pg-promise": "=5.9.3",
+    "pg-promise": "=6.3.4",
```

Comment:
change it to 6.3.5, there was a nasty bug fixed: https://github.com/vitaly-t/pg-promise/releases/tag/v.6.3.5